project/ProjectPlugin.scala

```diff
@@ -12,6 +12,20 @@ object ProjectPlugin extends AutoPlugin {
 
   override def requires: Plugins = plugins.JvmPlugin && OrgPoliciesPlugin
 
+  object autoImport {
+
+    lazy val V = new {
+      val scala212: String            = "2.12.10"
+      val cats: String = "2.0.0"
```

Comment:
Bad formatting

Reply:
That val shouldn't be there, deleting it...